.changeset/swift-steaks-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": docs
```

Comment:
Eek! This has to be one of major, minor or patch. When I said the change should be docs, I meant in the description below.

Reply:
Oh duh 🤦 . My autopilot was fully engaged on this one. Shall I make a PR to fix up the changeset?

Reply:
No worries. All sorted in the release that just went out. Also I have added some validation to prevent this now: 

https://github.com/cloudflare/workers-sdk/blob/main/.github/workflows/pullrequests.yml#L66-L67